packages/ui/src/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import type { SVGComponent } from "@eisbuk/svg";
+
+interface TabItemProps extends React.HTMLAttributes<HTMLButtonElement> {
+  label: string;
+  Icon: SVGComponent;
+  active?: boolean;
+}
+
+const TabItem: React.FC<TabItemProps> = ({
+  label,
+  Icon,
+  onClick,
+  active = false,
+  ...props
+}) => {
+  const onClickHandler = onClick ? onClick : () => {};
+
+  const buttonStateClasses = active
+    ? buttonActiveClasses
+    : buttonDefaultClasses;
+
+  const svgClassNames = svgClasses.join(" ");
+  const buttonClassNames = [...buttonBaseClasses, ...buttonStateClasses].join(
+    " "
+  );
+
+  return (
+    <button className={buttonClassNames} onClick={onClickHandler} {...props}>
+      <span className={svgClassNames}>
+        <Icon />
+      </span>
+      <span>{label}</span>
+    </button>
+  );
+};
+
+const svgClasses = ["w-6", "h-6"];
+
+const buttonBaseClasses = [
+  "flex",
+  "flex-col",
+  "sm:flex-row",
```

Comment:
We didn't specify breakpoints in the mockup (neither did Alex). I think this could go up to `md`, but let me know your opinion.

Reply:
Aaand yeah...I've just checked the breakpoints and I think it's safe to move this to `768px +`. So that it's still square(ish) on mobile for easier button touch.

Reply:
Ok, agreed - I will update